dozer-tests/Cargo.toml

```diff
@@ -14,6 +14,10 @@ path = "src/dozer_tests.rs"
 name = "dozer-test-client"
 path = "src/dozer_test_client.rs"
 
+[[bin]]
+name = "dozer-sqllogictest"
+path = "src/sqllogictest/src/main.rs"
+
 [dependencies]
```

Comment:
Considering we are building a binary, can we restrict the dependencies with a certain feature flag ? So we can build it separately ?

Reply:
It doesn't need many dependencies, so maybe we don't need to use a certain feature flag to restrict it